Source/WebKit/UIProcess/WebAuthentication/WebAuthenticatorCoordinatorProxy.h

```diff
@@ -81,11 +72,11 @@ struct WebAuthenticationRequestData;
 using CapabilitiesCompletionHandler = CompletionHandler<void(Vector<KeyValuePair<String, bool>>&&)>;
 using RequestCompletionHandler = CompletionHandler<void(const WebCore::AuthenticatorResponseData&, WebCore::AuthenticatorAttachment, const WebCore::ExceptionData&)>;
 
-class WebAuthenticatorCoordinatorProxy : public IPC::MessageReceiver {
+class WebAuthenticatorCoordinatorProxy : public IPC::MessageReceiver, public RefCounted<WebAuthenticatorCoordinatorProxy> {
```

Comment:
`WeakRef<WebPageProxy> m_webPageProxy;` is no longer safe after making this class RefCounted.

Reply:
Oh, right. Will fix.